package.json

```diff
@@ -10,8 +10,13 @@
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
   },
   "dependencies": {
+    "@react-navigation/native": "^6.0.13",
+    "@react-navigation/native-stack": "^6.9.0",
+    "@react-navigation/stack": "^6.3.2",
     "react": "18.1.0",
-    "react-native": "0.70.1"
+    "react-native": "0.70.1",
+    "react-native-safe-area-context": "^4.4.1",
+    "react-native-screens": "^3.18.0"
```

Comment:
Do we need these 2 dependencies right now? I don't see them being used currently in the code.

Reply:
https://reactnavigation.org/docs/getting-started/

Yes actually the navigation required these 2 packages so that stack or other navigations work.